language/tools/move-package/src/source_package/parsed_manifest.rs

```diff
@@ -43,6 +44,7 @@ pub struct Dependency {
     pub version: Option<Version>,
     pub digest: Option<PackageDigest>,
     pub git_info: Option<GitInfo>,
+    pub node_info: Option<NodeInfo>,
```

Comment:
Are these two mutually exclusive? Should it be something like `external_info`?

Reply:
Very different